### ISSUE TITLE
Is .net sdk 5 preview 1 supports blazor?

### ISSUE THREAD

**Issue by kmuthukumar23** [dotnet/aspnetcore#20093]
Hi guys,

Thanks in advance.

I have one doubt reg latest sdk 5.0 preview 1. Is this supports our blazor?

I have migrate the exsisting blazor server app to 5.0. but i have faced the following error.

![image](https://user-images.githubusercontent.com/31958591/77387611-e3676880-6db3-11ea-8ff4-6fe43da72120.png)

Can any one please help me on this if i am any thing missed on my side.  

I have used VS2019 16.5.0 and sdk 5.0 preview 1

**Comment by boukenka** [comment]
The answer is no, if I'm not mistaken. Please refer to the chapter "_Version updated to 3.2_" for more details on the official ASP.NET blog : [Blazor WebAssembly 3.2.0 Preview 1 release now available](https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-preview-1-release-now-available/)

**Comment by javiercn** [comment]
@kmuthukumar23 thanks for contacting us.

@boukenka is correct, we haven't done any work yet to support 5.0 specifically.

**Comment by MarkStega** [comment]
@javiercn  This was my response to @kmuthukumar23 on the gitter channel:

 I had a rocky upgrade from 3.2P2 to 5.0P1; I first installed the sdk & then opened my existing solution in VS 16.6P1 and none of the projects would open. I backed out the SDK and reopened my solution. I then used NuGet to update packages to 5.0P1. Then I reinstalled the SDK & runtime. After all of that I could build and run my solution in either SSB or CSB. However, VS no longer shows a client side option for a new Blazor project. So even though it was functional, I decided to revert to 3.2P2 since that line of code will be released in the May timeframe while 5.0 will not be ready until fall; I have to ship product before then. I am currently shipping and supporting a SSB version of my product and I'd like to move to CSB as soon as I can. If you are just starting development, don't need to ship before fall, and are willing to develop in SSB it appears that choosing the 5.0 pre-release train is a viable option. Of course, changes made in the 3.2 train may well be lagging in the 5.0 side.

**Comment by mkArtakMSFT** [comment]
@kmuthukumar23 please make sure that you haven't pinned the SDK. The Blazor Server projects should be working with .NET 5.0 preview SDKs.  If you're still seeing issues after installed the latest .NET 5.0 SDK, consider filing a new issue in the `dotnet/runtime` repo.
